components/contribution-flow/index.js

```diff
@@ -680,7 +681,16 @@ class ContributionFlow extends React.Component {
                       currency={currency}
                     />
                   </Box>
+                  <Box textAlign="center" mt={5}>
+                    <Link href={`/${collective.slug}`}>
+                      <FormattedMessage
+                        id="ContributionFlow.backToCollectivePage"
+                        defaultMessage="Back to Collective Page"
```

Comment:
This one should also not be displayed on embeds. The underlying policy is: people should not be redirected anywhere else than the contribution flow when using the embed version.

Reply:
Handled this as well. 😄 Thanks for the review.